### PR TITLE
update and upgrade with some tests added and an action for the tests, also I did a `deno fmt`

### PR DESCRIPTION
**Pull request by digitaldesigndj**
Quick and dirty update/upgrade. To the latest Fresh also std/cookies and some of the std imports. and more...

### CODE REVIEW

**Comment by digitaldesigndj** [comment]
I have also added the start of a test case complete with github action.

https://github.com/digitaldesigndj/fresh-session/actions

I also added a cookie jar to fetch to make the testing work properly

File: tests/fixture/routes/dashboard/index.tsx
Comment (digitaldesigndj) [inline tests/fixture/routes/dashboard/index.tsx:1]: Want to make sure this is somewhere else before it goes

**Review by digitaldesigndj — COMMENTED** [review]
Import Maps updated to new format and upgraded fresh with dependencies too.

Added tests for Fixture and the KV Store example

File: tests/fixture/routes/dashboard/index.tsx
Comment (digitaldesigndj) [reply tests/fixture/routes/dashboard/index.tsx:1]: Its in the readme

**Comment by digitaldesigndj** [comment]
That wasn't quick after all, but oh man it's so dirty.

File: example/use_kv_store/deno.json [hunk 1]
```diff
@@ -2,9 +2,20 @@
...
+  "imports": {
+    "$fresh/": "https://deno.land/x/fresh@1.3.1/",
+    "preact": "https://esm.sh/preact@10.15.1",
+    "preact/": "https://esm.sh/preact@10.15.1/",
+    "preact-render-to-string": "https://esm.sh/*preact-render-to-string@6.1.0",
+    "@preact/signals": "https://esm.sh/*@preact/signals@1.0.3",
+    "@preact/signals-core": "https://esm.sh/*@preact/signals-core@1.0.1",
+    "twind": "https://esm.sh/twind@0.16.17",
+    "twind/": "https://esm.sh/twind@0.16.17/",
+    "fresh-session/": "https://raw.githubusercontent.com/Octo8080X/fresh-session/main/"
```
Comment (digitaldesigndj) [inline example/use_kv_store/deno.json:18]: This import makes stuff work until the 0.2.2 release is cut, there are 2
See Issue #28 

```
"fresh-session/": "https://deno.land/x/fresh-session@0.2.2/"
```

File: example/use_redis_store/deno.json [hunk 2]
```diff
@@ -2,9 +2,19 @@
...
+  },
+  "imports": {
+    "$fresh/": "https://deno.land/x/fresh@1.3.1/",
+    "preact": "https://esm.sh/preact@10.15.1",
+    "preact/": "https://esm.sh/preact@10.15.1/",
+    "preact-render-to-string": "https://esm.sh/*preact-render-to-string@6.1.0",
+    "@preact/signals": "https://esm.sh/*@preact/signals@1.0.3",
+    "@preact/signals-core": "https://esm.sh/*@preact/signals-core@1.0.1",
+    "redis/": "https://deno.land/x/redis@v0.25.0/",
+    "fresh-session/": "https://raw.githubusercontent.com/Octo8080X/fresh-session/main/"
```
Comment (digitaldesigndj) [inline example/use_redis_store/deno.json]: It's the other one.
Use this when Issue #28 resolves 
```
"fresh-session/": "https://deno.land/x/fresh-session@0.2.2/"
```

Comment (digitaldesigndj) [reply example/use_redis_store/deno.json]: Changed this because we dont need the updated version

**Comment by Octo8080X** [comment]
@digitaldesigndj  san

We appreciate your contribution.
Please give me some time today or tomorrow so I can check.

Thank you.

**Review by Octo8080X — APPROVED** [review]

**Pull request merged**